flink-table/flink-sql-client/src/test/resources/sql/function.q

```diff
@@ -145,9 +170,14 @@ drop function if exists non_func;
 
 # should contain func11, not contain func10
 show user functions;
-func11
-func3
-func4
++---------------+
+| function name |
++---------------+
+|         func4 |
+|        func11 |
+|         func3 |
++---------------+
```

Comment:
We still need a sorted result, but the sort should be supported by TableEnvironment. Could you sort the result for `TableEnvironment#listFunctions` and `listUserDefinedFunctions`?